pkg/engine/lifeycletest/test_plan.go

```diff
@@ -96,12 +96,12 @@ func (op TestOp) RunWithContext(
 	_, res := op(info, ctx, opts, dryRun)
```

Comment:
Do we need to check `res`? We didn't before, but it seems like we should.

Reply:
No we feed it directly into validate which should check it (or validate is nil in which case we return it)